### ISSUE TITLE
no vacuum to select

### ISSUE THREAD

**Issue by tuupos** [CodeFoodPixels/robovac#97]
![image](https://github.com/user-attachments/assets/d708f2b3-82c9-440d-954d-4c83237c07e4)

**Comment by tuupos** [comment]
Ive got the eufy robovac 11c pet edition. Maybe it isn t supported?

**Comment by rtymchyk** [comment]
Same issue with the X10 Pro so probably not just you

**Comment by mattmichaels** [comment]
following - nothing to select here either. own a pair of x10 pros

**Comment by Maschine20** [comment]
following - same issue for the Eufy X10 pro

**Comment by aodtcr** [comment]
following - also with X10 Pro

**Comment by Jango024** [comment]
following - also with X10 Pro

**Comment by rstuhlmuller** [comment]
following - also with X10 Pro

**Comment by edrager76** [comment]
I've been unable to use both my old 30C and my new L60 with this integration.

Up to like a year ago (maybe less?) everything worked fine on my 30C.

I'm able to download the integration in HACS, then add it in integrations.
Can login without issues, it sees my 2 vacuums, I can click configure, it detects the IP fine, and then I only see an unavailable battery status, nothing else.

Have tried installing a brand new (for the rest empty) home assistant instance in proxmox; same result.

I do see that in the integration page, the version number is 1.0.0 (so in http://<ha-url>:8123/config/integrations/integration/robovac).

I seem to be unable to update this at all.

Is that normal, or not?

**Comment by dcbell** [comment]
> Posting here because unable to create a new issue.
> 
> I've been unable to use both my old 30C and my new L60 with this integration.
> 
> Up to like a year ago (maybe less?) everything worked fine on my 30C.
> 
> I'm able to download the integration in HACS, then add it in integrations. Can login without issues, it sees my 2 vacuums, I can click configure, it detects the IP fine, and then I only see an unavailable battery status, nothing else.
> 
> Have tried installing a brand new (for the rest empty) home assistant instance in proxmox; same result.
> 
> I do see that in the integration page, the version number is 1.0.0 (so in http://:8123/config/integrations/integration/robovac).
> 
> I seem to be unable to update this at all.
> 
> Is that normal, or not?

did you find any solution? I just tried to add my L60 and I have the same issue. I can select my vacuum and set the IP, but then just an unavailable battery status.

**Comment by edrager76** [comment]
> did you find any solution? I just tried to add my L60 and I have the same issue. I can select my vacuum and set the IP, but then just an unavailable battery status.

Nope. Seems like the integration is just broken somehow, at least for me. I am now using Alexa routines to control the vacuum, using helpers created in HA, published to Alexa. They function OK as starter.

**Comment by saeraz** [comment]
following - also with X10 Pro (12.3.2025)

**Comment by Sheriff867** [comment]
following - also with X10 Pro

**Comment by kieranbehan** [comment]
Following for me - x10 Omni pro - same issue - no vacuum available
